mne/viz/ui_events.py

```diff
@@ -347,10 +347,8 @@ def link(fig1, fig2, *, include_events=None, exclude_events=None, verbose=None):
 
     Parameters
     ----------
-    fig1 : matplotlib.figure.Figure | Figure3D
-        The first figure whose event channel will be linked to the second.
-    fig2 : matplotlib.figure.Figure | Figure3D
-        The second figure whose event channel will be linked to the first.
+    figs : matplotlib.figure.Figure | Figure3D
```

Comment:
I think this should be something like
```suggestion
    figs : tuple of matplotlib.figure.Figure, shape (n,) | tuple of Figure3D, shape (n,)
```